src/everything/everything.ts

```diff
@@ -648,6 +663,42 @@ export const createServer = () => {
       return { content };
     }
 
+    if (name === ToolName.GET_RESOURCE_LINKS) {
+      const { count } = GetResourceLinksSchema.parse(args);
+      const content = [];
+
+      // Add intro text
+      content.push({
+        type: "text",
+        text: `Here are ${count} resource links to resources available in this server (see full output in tool response if your client does not support resource_link yet):`,
+      });
+
+      // Return resource links to actual resources from ALL_RESOURCES
+      const actualCount = Math.min(count, ALL_RESOURCES.length);
+      for (let i = 0; i < actualCount; i++) {
+        const resource = ALL_RESOURCES[i];
+        content.push({
+          type: "resource_link",
+          uri: resource.uri,
+          name: resource.name,
+          description: `Resource ${i + 1}: ${
+            resource.mimeType === "text/plain"
+              ? "plaintext resource"
+              : "binary blob resource"
+          }`,
+          mimeType: resource.mimeType,
+        });
+      }
+
+      // Add summary text
+      content.push({
+        type: "text",
+        text: `\nThese resource links reference actual resources available through the resources/read endpoint.`,
```

Comment:
Why the leading newline?
<img width="753" alt="Screenshot 2025-07-04 at 1 36 29 PM" src="https://github.com/user-attachments/assets/62c76e90-d04b-41d6-b445-0d6ef0060342" />

Reply:
And is this final text content even necessary?